src/Pages/Checkout/Checkout.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import Shipping from './Component/Shipping/Shipping';
+import Payment from './Component/Payment/Payment';
+import { BASE_URL } from '../../config.js';
+import { Token } from '../../config.js';
+import Order from './Component/Order/Order';
+import './Checkout.scss';
+
+//부모:마이박스에서 섭토탈, 토탈 받아오기
+const Checkout = ({ subtotal, total }) => {
+  const [orderInfo, setOrderInfo] = useState({
+    zipCode: null,
+    addressMain: null,
+    addressDetail: null,
+    updateAddress: null,
+    wayOfShipping: null,
+    wayOfPayment: null,
+  });
+
+  const orderInfoHandler = e => {
+    setOrderInfo({ ...orderInfo, [e.target.className]: e.target.value });
+  };
+
+  const patchAddress = () => {
+    fetch(`${BASE_URL}/users/`, {
+      method: 'PATCH',
+      headers: {
+        // Authorization: localStorage.getItem(토큰),
+        Authorization: Token,
+      },
+      body: JSON.stringify({
+        zip_code: orderInfo.zipCode,
+        location: orderInfo.addressMain,
+        datail_address: orderInfo.addressDetail,
+      }),
+    })
+      .then(response => {
+        if (response.ok) {
+          alert('주소변경이 완료되었습니다.');
+          window.location.reload();
+        } else {
+          throw new Error('Unexpected Error');
+        }
+      })
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.log(error.message);
+      });
+  };
+
+  const postBalance = () => {
+    fetch(`${BASE_URL}/orders`, {
+      method: 'POST',
+      headers: {
+        // Authorization: localStorage.getItem(토큰),
+        Authorization: Token,
+      },
+      body: JSON.stringify({
+        total_price: subtotal,
+      }),
+    })
+      .then(response => {
+        if (response.ok) {
+          alert('주문이 완료되었습니다.');
+        } else if (response.message === 'moneyless') {
+          alert('잔액이 부족합니다...안타깝습니다..');
+        } else if (response.message === 'stockless') {
+          alert('재고가 부족합니다...분발하겠습니다..');
+        } else {
+          throw new Error('Unexpected Error');
+        }
+      })
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.log(error.message);
+      });
+  };
+
+  const submitOrder = () => {
+    if (
+      orderInfo.zipCode &&
+      orderInfo.addressDetail &&
+      orderInfo.wayOfShipping &&
+      orderInfo.wayOfPayment
+    ) {
+      if (orderInfo.updateAddress === true) {
```

Comment:
```suggestion
      if (orderInfo.updateAddress) {
```
plus 스트링 값에 !! 주면 불린값으로 사용할 수 있습니다!
```jsx
let hello = 'jaewoong'
console.log(hello) //'jaewoong'
console.log(!hello) //false
console.log(!!hello) //true
```

Reply:
수정하였습니다.! 예시 감사합니다